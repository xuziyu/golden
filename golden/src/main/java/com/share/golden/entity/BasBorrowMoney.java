package com.share.golden.entity;

public class BasBorrowMoney {
    private String borrowId;

    private String borrowSerial;

    private String userId;

    private String borrowCompany;

    private String borrowMoney;

    private String borrowRate;

    private String timeLimit;

    private String borrowLimit;

    private String borrowState;

    private String borrowRecommend;

    private String borrowType;

    private String borrowRemark1;

    private String borrowRemark2;

    public String getBorrowId() {
        return borrowId;
    }

    public void setBorrowId(String borrowId) {
        this.borrowId = borrowId == null ? null : borrowId.trim();
    }

    public String getBorrowSerial() {
        return borrowSerial;
    }

    public void setBorrowSerial(String borrowSerial) {
        this.borrowSerial = borrowSerial == null ? null : borrowSerial.trim();
    }

    public String getUserId() {
        return userId;
    }

    public void setUserId(String userId) {
        this.userId = userId == null ? null : userId.trim();
    }

    public String getBorrowCompany() {
        return borrowCompany;
    }

    public void setBorrowCompany(String borrowCompany) {
        this.borrowCompany = borrowCompany == null ? null : borrowCompany.trim();
    }

    public String getBorrowMoney() {
        return borrowMoney;
    }

    public void setBorrowMoney(String borrowMoney) {
        this.borrowMoney = borrowMoney == null ? null : borrowMoney.trim();
    }

    public String getBorrowRate() {
        return borrowRate;
    }

    public void setBorrowRate(String borrowRate) {
        this.borrowRate = borrowRate == null ? null : borrowRate.trim();
    }

    public String getTimeLimit() {
        return timeLimit;
    }

    public void setTimeLimit(String timeLimit) {
        this.timeLimit = timeLimit == null ? null : timeLimit.trim();
    }

    public String getBorrowLimit() {
        return borrowLimit;
    }

    public void setBorrowLimit(String borrowLimit) {
        this.borrowLimit = borrowLimit == null ? null : borrowLimit.trim();
    }

    public String getBorrowState() {
        return borrowState;
    }

    public void setBorrowState(String borrowState) {
        this.borrowState = borrowState == null ? null : borrowState.trim();
    }

    public String getBorrowRecommend() {
        return borrowRecommend;
    }

    public void setBorrowRecommend(String borrowRecommend) {
        this.borrowRecommend = borrowRecommend == null ? null : borrowRecommend.trim();
    }

    public String getBorrowType() {
        return borrowType;
    }

    public void setBorrowType(String borrowType) {
        this.borrowType = borrowType == null ? null : borrowType.trim();
    }

    public String getBorrowRemark1() {
        return borrowRemark1;
    }

    public void setBorrowRemark1(String borrowRemark1) {
        this.borrowRemark1 = borrowRemark1 == null ? null : borrowRemark1.trim();
    }

    public String getBorrowRemark2() {
        return borrowRemark2;
    }

    public void setBorrowRemark2(String borrowRemark2) {
        this.borrowRemark2 = borrowRemark2 == null ? null : borrowRemark2.trim();
    }
}