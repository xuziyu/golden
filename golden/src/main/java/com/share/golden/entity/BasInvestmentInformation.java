package com.share.golden.entity;

import java.math.BigDecimal;
import java.util.Date;

public class BasInvestmentInformation {
    private String investorId;

    private String userId;

    private String tenderId;

    private BigDecimal investorMoney;

    private String investorStatus;

    private String investorStyle;

    private String brrowStatus;

    private String investorRest;

    private String profitModel;

    private BigDecimal profitMoney;

    private Date investorDate;

    private String replayDate;

    private String investorType;

    public String getInvestorId() {
        return investorId;
    }

    public void setInvestorId(String investorId) {
        this.investorId = investorId == null ? null : investorId.trim();
    }

    public String getUserId() {
        return userId;
    }

    public void setUserId(String userId) {
        this.userId = userId == null ? null : userId.trim();
    }

    public String getTenderId() {
        return tenderId;
    }

    public void setTenderId(String tenderId) {
        this.tenderId = tenderId == null ? null : tenderId.trim();
    }

    public BigDecimal getInvestorMoney() {
        return investorMoney;
    }

    public void setInvestorMoney(BigDecimal investorMoney) {
        this.investorMoney = investorMoney;
    }

    public String getInvestorStatus() {
        return investorStatus;
    }

    public void setInvestorStatus(String investorStatus) {
        this.investorStatus = investorStatus == null ? null : investorStatus.trim();
    }

    public String getInvestorStyle() {
        return investorStyle;
    }

    public void setInvestorStyle(String investorStyle) {
        this.investorStyle = investorStyle == null ? null : investorStyle.trim();
    }

    public String getBrrowStatus() {
        return brrowStatus;
    }

    public void setBrrowStatus(String brrowStatus) {
        this.brrowStatus = brrowStatus == null ? null : brrowStatus.trim();
    }

    public String getInvestorRest() {
        return investorRest;
    }

    public void setInvestorRest(String investorRest) {
        this.investorRest = investorRest == null ? null : investorRest.trim();
    }

    public String getProfitModel() {
        return profitModel;
    }

    public void setProfitModel(String profitModel) {
        this.profitModel = profitModel == null ? null : profitModel.trim();
    }

    public BigDecimal getProfitMoney() {
        return profitMoney;
    }

    public void setProfitMoney(BigDecimal profitMoney) {
        this.profitMoney = profitMoney;
    }

    public Date getInvestorDate() {
        return investorDate;
    }

    public void setInvestorDate(Date investorDate) {
        this.investorDate = investorDate;
    }

    public String getReplayDate() {
        return replayDate;
    }

    public void setReplayDate(String replayDate) {
        this.replayDate = replayDate == null ? null : replayDate.trim();
    }

    public String getInvestorType() {
        return investorType;
    }

    public void setInvestorType(String investorType) {
        this.investorType = investorType == null ? null : investorType.trim();
    }
}