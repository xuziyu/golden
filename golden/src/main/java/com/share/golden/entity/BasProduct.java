package com.share.golden.entity;

import java.math.BigDecimal;
import java.util.Date;

public class BasProduct {
    private String productId;

    private BigDecimal productMoney;

    private String productIncome;

    private Date productTime;

    private String productType;

    private String productWay;

    private Date pcount;

    private String productProgress;

    private String productSaveway;

    private String productRateben;

    private Date productPublishTime;

    private String productName;

    private BigDecimal productTotalMoney;

    private String investmentScope;

    private String useOfProceeds;

    private String productState;

    private String productPicture;

    private String produceId;

    private String productDesc;

    private String productSafe;

    public String getProductId() {
        return productId;
    }

    public void setProductId(String productId) {
        this.productId = productId == null ? null : productId.trim();
    }

    public BigDecimal getProductMoney() {
        return productMoney;
    }

    public void setProductMoney(BigDecimal productMoney) {
        this.productMoney = productMoney;
    }

    public String getProductIncome() {
        return productIncome;
    }

    public void setProductIncome(String productIncome) {
        this.productIncome = productIncome == null ? null : productIncome.trim();
    }

    public Date getProductTime() {
        return productTime;
    }

    public void setProductTime(Date productTime) {
        this.productTime = productTime;
    }

    public String getProductType() {
        return productType;
    }

    public void setProductType(String productType) {
        this.productType = productType == null ? null : productType.trim();
    }

    public String getProductWay() {
        return productWay;
    }

    public void setProductWay(String productWay) {
        this.productWay = productWay == null ? null : productWay.trim();
    }

    public Date getPcount() {
        return pcount;
    }

    public void setPcount(Date pcount) {
        this.pcount = pcount;
    }

    public String getProductProgress() {
        return productProgress;
    }

    public void setProductProgress(String productProgress) {
        this.productProgress = productProgress == null ? null : productProgress.trim();
    }

    public String getProductSaveway() {
        return productSaveway;
    }

    public void setProductSaveway(String productSaveway) {
        this.productSaveway = productSaveway == null ? null : productSaveway.trim();
    }

    public String getProductRateben() {
        return productRateben;
    }

    public void setProductRateben(String productRateben) {
        this.productRateben = productRateben == null ? null : productRateben.trim();
    }

    public Date getProductPublishTime() {
        return productPublishTime;
    }

    public void setProductPublishTime(Date productPublishTime) {
        this.productPublishTime = productPublishTime;
    }

    public String getProductName() {
        return productName;
    }

    public void setProductName(String productName) {
        this.productName = productName == null ? null : productName.trim();
    }

    public BigDecimal getProductTotalMoney() {
        return productTotalMoney;
    }

    public void setProductTotalMoney(BigDecimal productTotalMoney) {
        this.productTotalMoney = productTotalMoney;
    }

    public String getInvestmentScope() {
        return investmentScope;
    }

    public void setInvestmentScope(String investmentScope) {
        this.investmentScope = investmentScope == null ? null : investmentScope.trim();
    }

    public String getUseOfProceeds() {
        return useOfProceeds;
    }

    public void setUseOfProceeds(String useOfProceeds) {
        this.useOfProceeds = useOfProceeds == null ? null : useOfProceeds.trim();
    }

    public String getProductState() {
        return productState;
    }

    public void setProductState(String productState) {
        this.productState = productState == null ? null : productState.trim();
    }

    public String getProductPicture() {
        return productPicture;
    }

    public void setProductPicture(String productPicture) {
        this.productPicture = productPicture == null ? null : productPicture.trim();
    }

    public String getProduceId() {
        return produceId;
    }

    public void setProduceId(String produceId) {
        this.produceId = produceId == null ? null : produceId.trim();
    }

    public String getProductDesc() {
        return productDesc;
    }

    public void setProductDesc(String productDesc) {
        this.productDesc = productDesc == null ? null : productDesc.trim();
    }

    public String getProductSafe() {
        return productSafe;
    }

    public void setProductSafe(String productSafe) {
        this.productSafe = productSafe == null ? null : productSafe.trim();
    }
}