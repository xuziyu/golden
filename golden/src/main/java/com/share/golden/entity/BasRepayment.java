package com.share.golden.entity;

import java.util.Date;

public class BasRepayment {
    private String repayId;

    private String repayCode;

    private Date repayDate;

    private String repayState;

    private String repayTime;

    private String repayRemark1;

    private String repayRemark2;

    public String getRepayId() {
        return repayId;
    }

    public void setRepayId(String repayId) {
        this.repayId = repayId == null ? null : repayId.trim();
    }

    public String getRepayCode() {
        return repayCode;
    }

    public void setRepayCode(String repayCode) {
        this.repayCode = repayCode == null ? null : repayCode.trim();
    }

    public Date getRepayDate() {
        return repayDate;
    }

    public void setRepayDate(Date repayDate) {
        this.repayDate = repayDate;
    }

    public String getRepayState() {
        return repayState;
    }

    public void setRepayState(String repayState) {
        this.repayState = repayState == null ? null : repayState.trim();
    }

    public String getRepayTime() {
        return repayTime;
    }

    public void setRepayTime(String repayTime) {
        this.repayTime = repayTime == null ? null : repayTime.trim();
    }

    public String getRepayRemark1() {
        return repayRemark1;
    }

    public void setRepayRemark1(String repayRemark1) {
        this.repayRemark1 = repayRemark1 == null ? null : repayRemark1.trim();
    }

    public String getRepayRemark2() {
        return repayRemark2;
    }

    public void setRepayRemark2(String repayRemark2) {
        this.repayRemark2 = repayRemark2 == null ? null : repayRemark2.trim();
    }
}