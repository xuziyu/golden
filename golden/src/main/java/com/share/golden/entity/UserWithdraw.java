package com.share.golden.entity;

import java.math.BigDecimal;
import java.util.Date;

public class UserWithdraw {
	private String withdrawId;

	private String withdrawAccount;

	private String withdrawBank;

	private BigDecimal withdrawMoney;

	private BigDecimal realMoney;

	private BigDecimal withdrawServiceCharge;

	private Date withdrawTime;

	private Date intoTime;

	private String withdrawState;

	private String auditorId;

	private String auditor;

	private Date auditTime;

	private String remark;

	public String getWithdrawId() {
		return withdrawId;
	}

	public void setWithdrawId(String withdrawId) {
		this.withdrawId = withdrawId == null ? null : withdrawId.trim();
	}

	public String getWithdrawAccount() {
		return withdrawAccount;
	}

	public void setWithdrawAccount(String withdrawAccount) {
		this.withdrawAccount = withdrawAccount == null ? null : withdrawAccount.trim();
	}

	public String getWithdrawBank() {
		return withdrawBank;
	}

	public void setWithdrawBank(String withdrawBank) {
		this.withdrawBank = withdrawBank == null ? null : withdrawBank.trim();
	}

	public BigDecimal getWithdrawMoney() {
		return withdrawMoney;
	}

	public void setWithdrawMoney(BigDecimal withdrawMoney) {
		this.withdrawMoney = withdrawMoney;
	}

	public BigDecimal getRealMoney() {
		return realMoney;
	}

	public void setRealMoney(BigDecimal realMoney) {
		this.realMoney = realMoney;
	}

	public BigDecimal getWithdrawServiceCharge() {
		return withdrawServiceCharge;
	}

	public void setWithdrawServiceCharge(BigDecimal withdrawServiceCharge) {
		this.withdrawServiceCharge = withdrawServiceCharge;
	}

	public Date getWithdrawTime() {
		return withdrawTime;
	}

	public void setWithdrawTime(Date withdrawTime) {
		this.withdrawTime = withdrawTime;
	}

	public Date getIntoTime() {
		return intoTime;
	}

	public void setIntoTime(Date intoTime) {
		this.intoTime = intoTime;
	}

	public String getWithdrawState() {
		return withdrawState;
	}

	public void setWithdrawState(String withdrawState) {
		this.withdrawState = withdrawState == null ? null : withdrawState.trim();
	}

	public String getAuditorId() {
		return auditorId;
	}

	public void setAuditorId(String auditorId) {
		this.auditorId = auditorId == null ? null : auditorId.trim();
	}

	public String getAuditor() {
		return auditor;
	}

	public void setAuditor(String auditor) {
		this.auditor = auditor == null ? null : auditor.trim();
	}

	public Date getAuditTime() {
		return auditTime;
	}

	public void setAuditTime(Date auditTime) {
		this.auditTime = auditTime;
	}

	public String getRemark() {
		return remark;
	}

	public void setRemark(String remark) {
		this.remark = remark == null ? null : remark.trim();
	}
}