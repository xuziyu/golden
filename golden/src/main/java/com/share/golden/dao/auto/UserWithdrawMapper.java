package com.share.golden.dao.auto;

import com.share.golden.entity.UserWithdraw;
import com.share.golden.entity.UserWithdrawExample;
import java.util.List;
import org.apache.ibatis.annotations.Param;

public interface UserWithdrawMapper {
    long countByExample(UserWithdrawExample example);

    int deleteByExample(UserWithdrawExample example);

    int deleteByPrimaryKey(String withdrawId);

    int insert(UserWithdraw record);

    int insertSelective(UserWithdraw record);

    List<UserWithdraw> selectByExample(UserWithdrawExample example);

    UserWithdraw selectByPrimaryKey(String withdrawId);

    int updateByExampleSelective(@Param("record") UserWithdraw record, @Param("example") UserWithdrawExample example);

    int updateByExample(@Param("record") UserWithdraw record, @Param("example") UserWithdrawExample example);

    int updateByPrimaryKeySelective(UserWithdraw record);

    int updateByPrimaryKey(UserWithdraw record);
}