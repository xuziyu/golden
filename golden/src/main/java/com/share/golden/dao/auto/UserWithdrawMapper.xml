<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE mapper PUBLIC "-//mybatis.org//DTD Mapper 3.0//EN" "http://mybatis.org/dtd/mybatis-3-mapper.dtd">
<mapper namespace="com.share.golden.dao.auto.UserWithdrawMapper">
  <resultMap id="BaseResultMap" type="com.share.golden.entity.UserWithdraw">
    <id column="WITHDRAW_ID" jdbcType="VARCHAR" property="withdrawId" />
    <result column="WITHDRAW_ACCOUNT" jdbcType="VARCHAR" property="withdrawAccount" />
    <result column="WITHDRAW_BANK" jdbcType="VARCHAR" property="withdrawBank" />
    <result column="WITHDRAW_MONEY" jdbcType="DECIMAL" property="withdrawMoney" />
    <result column="REAL_MONEY" jdbcType="DECIMAL" property="realMoney" />
    <result column="WITHDRAW_SERVICE_CHARGE" jdbcType="DECIMAL" property="withdrawServiceCharge" />
    <result column="WITHDRAW_TIME" jdbcType="TIMESTAMP" property="withdrawTime" />
    <result column="INTO_TIME" jdbcType="TIMESTAMP" property="intoTime" />
    <result column="WITHDRAW_STATE" jdbcType="VARCHAR" property="withdrawState" />
    <result column="AUDITOR_ID" jdbcType="VARCHAR" property="auditorId" />
    <result column="AUDITOR" jdbcType="VARCHAR" property="auditor" />
    <result column="AUDIT_TIME" jdbcType="TIMESTAMP" property="auditTime" />
    <result column="REMARK" jdbcType="VARCHAR" property="remark" />
  </resultMap>
  <sql id="Example_Where_Clause">
    <where>
      <foreach collection="oredCriteria" item="criteria" separator="or">
        <if test="criteria.valid">
          <trim prefix="(" prefixOverrides="and" suffix=")">
            <foreach collection="criteria.criteria" item="criterion">
              <choose>
                <when test="criterion.noValue">
                  and ${criterion.condition}
                </when>
                <when test="criterion.singleValue">
                  and ${criterion.condition} #{criterion.value}
                </when>
                <when test="criterion.betweenValue">
                  and ${criterion.condition} #{criterion.value} and #{criterion.secondValue}
                </when>
                <when test="criterion.listValue">
                  and ${criterion.condition}
                  <foreach close=")" collection="criterion.value" item="listItem" open="(" separator=",">
                    #{listItem}
                  </foreach>
                </when>
              </choose>
            </foreach>
          </trim>
        </if>
      </foreach>
    </where>
  </sql>
  <sql id="Update_By_Example_Where_Clause">
    <where>
      <foreach collection="example.oredCriteria" item="criteria" separator="or">
        <if test="criteria.valid">
          <trim prefix="(" prefixOverrides="and" suffix=")">
            <foreach collection="criteria.criteria" item="criterion">
              <choose>
                <when test="criterion.noValue">
                  and ${criterion.condition}
                </when>
                <when test="criterion.singleValue">
                  and ${criterion.condition} #{criterion.value}
                </when>
                <when test="criterion.betweenValue">
                  and ${criterion.condition} #{criterion.value} and #{criterion.secondValue}
                </when>
                <when test="criterion.listValue">
                  and ${criterion.condition}
                  <foreach close=")" collection="criterion.value" item="listItem" open="(" separator=",">
                    #{listItem}
                  </foreach>
                </when>
              </choose>
            </foreach>
          </trim>
        </if>
      </foreach>
    </where>
  </sql>
  <sql id="Base_Column_List">
    "WITHDRAW_ID", "WITHDRAW_ACCOUNT", "WITHDRAW_BANK", "WITHDRAW_MONEY", "REAL_MONEY", 
    "WITHDRAW_SERVICE_CHARGE", "WITHDRAW_TIME", "INTO_TIME", "WITHDRAW_STATE", "AUDITOR_ID", 
    "AUDITOR", "AUDIT_TIME", "REMARK"
  </sql>
  <select id="selectByExample" parameterType="com.share.golden.entity.UserWithdrawExample" resultMap="BaseResultMap">
    select
    <if test="distinct">
      distinct
    </if>
    <include refid="Base_Column_List" />
    from USER_WITHDRAW
    <if test="_parameter != null">
      <include refid="Example_Where_Clause" />
    </if>
    <if test="orderByClause != null">
      order by ${orderByClause}
    </if>
  </select>
  <select id="selectByPrimaryKey" parameterType="java.lang.String" resultMap="BaseResultMap">
    select 
    <include refid="Base_Column_List" />
    from USER_WITHDRAW
    where "WITHDRAW_ID" = #{withdrawId,jdbcType=VARCHAR}
  </select>
  <delete id="deleteByPrimaryKey" parameterType="java.lang.String">
    delete from USER_WITHDRAW
    where "WITHDRAW_ID" = #{withdrawId,jdbcType=VARCHAR}
  </delete>
  <delete id="deleteByExample" parameterType="com.share.golden.entity.UserWithdrawExample">
    delete from USER_WITHDRAW
    <if test="_parameter != null">
      <include refid="Example_Where_Clause" />
    </if>
  </delete>
  <insert id="insert" parameterType="com.share.golden.entity.UserWithdraw">
    insert into USER_WITHDRAW ("WITHDRAW_ID", "WITHDRAW_ACCOUNT", "WITHDRAW_BANK", 
      "WITHDRAW_MONEY", "REAL_MONEY", "WITHDRAW_SERVICE_CHARGE", 
      "WITHDRAW_TIME", "INTO_TIME", "WITHDRAW_STATE", 
      "AUDITOR_ID", "AUDITOR", "AUDIT_TIME", 
      "REMARK")
    values (#{withdrawId,jdbcType=VARCHAR}, #{withdrawAccount,jdbcType=VARCHAR}, #{withdrawBank,jdbcType=VARCHAR}, 
      #{withdrawMoney,jdbcType=DECIMAL}, #{realMoney,jdbcType=DECIMAL}, #{withdrawServiceCharge,jdbcType=DECIMAL}, 
      #{withdrawTime,jdbcType=TIMESTAMP}, #{intoTime,jdbcType=TIMESTAMP}, #{withdrawState,jdbcType=VARCHAR}, 
      #{auditorId,jdbcType=VARCHAR}, #{auditor,jdbcType=VARCHAR}, #{auditTime,jdbcType=TIMESTAMP}, 
      #{remark,jdbcType=VARCHAR})
  </insert>
  <insert id="insertSelective" parameterType="com.share.golden.entity.UserWithdraw">
    insert into USER_WITHDRAW
    <trim prefix="(" suffix=")" suffixOverrides=",">
      <if test="withdrawId != null">
        "WITHDRAW_ID",
      </if>
      <if test="withdrawAccount != null">
        "WITHDRAW_ACCOUNT",
      </if>
      <if test="withdrawBank != null">
        "WITHDRAW_BANK",
      </if>
      <if test="withdrawMoney != null">
        "WITHDRAW_MONEY",
      </if>
      <if test="realMoney != null">
        "REAL_MONEY",
      </if>
      <if test="withdrawServiceCharge != null">
        "WITHDRAW_SERVICE_CHARGE",
      </if>
      <if test="withdrawTime != null">
        "WITHDRAW_TIME",
      </if>
      <if test="intoTime != null">
        "INTO_TIME",
      </if>
      <if test="withdrawState != null">
        "WITHDRAW_STATE",
      </if>
      <if test="auditorId != null">
        "AUDITOR_ID",
      </if>
      <if test="auditor != null">
        "AUDITOR",
      </if>
      <if test="auditTime != null">
        "AUDIT_TIME",
      </if>
      <if test="remark != null">
        "REMARK",
      </if>
    </trim>
    <trim prefix="values (" suffix=")" suffixOverrides=",">
      <if test="withdrawId != null">
        #{withdrawId,jdbcType=VARCHAR},
      </if>
      <if test="withdrawAccount != null">
        #{withdrawAccount,jdbcType=VARCHAR},
      </if>
      <if test="withdrawBank != null">
        #{withdrawBank,jdbcType=VARCHAR},
      </if>
      <if test="withdrawMoney != null">
        #{withdrawMoney,jdbcType=DECIMAL},
      </if>
      <if test="realMoney != null">
        #{realMoney,jdbcType=DECIMAL},
      </if>
      <if test="withdrawServiceCharge != null">
        #{withdrawServiceCharge,jdbcType=DECIMAL},
      </if>
      <if test="withdrawTime != null">
        #{withdrawTime,jdbcType=TIMESTAMP},
      </if>
      <if test="intoTime != null">
        #{intoTime,jdbcType=TIMESTAMP},
      </if>
      <if test="withdrawState != null">
        #{withdrawState,jdbcType=VARCHAR},
      </if>
      <if test="auditorId != null">
        #{auditorId,jdbcType=VARCHAR},
      </if>
      <if test="auditor != null">
        #{auditor,jdbcType=VARCHAR},
      </if>
      <if test="auditTime != null">
        #{auditTime,jdbcType=TIMESTAMP},
      </if>
      <if test="remark != null">
        #{remark,jdbcType=VARCHAR},
      </if>
    </trim>
  </insert>
  <select id="countByExample" parameterType="com.share.golden.entity.UserWithdrawExample" resultType="java.lang.Long">
    select count(*) from USER_WITHDRAW
    <if test="_parameter != null">
      <include refid="Example_Where_Clause" />
    </if>
  </select>
  <update id="updateByExampleSelective" parameterType="map">
    update USER_WITHDRAW
    <set>
      <if test="record.withdrawId != null">
        "WITHDRAW_ID" = #{record.withdrawId,jdbcType=VARCHAR},
      </if>
      <if test="record.withdrawAccount != null">
        "WITHDRAW_ACCOUNT" = #{record.withdrawAccount,jdbcType=VARCHAR},
      </if>
      <if test="record.withdrawBank != null">
        "WITHDRAW_BANK" = #{record.withdrawBank,jdbcType=VARCHAR},
      </if>
      <if test="record.withdrawMoney != null">
        "WITHDRAW_MONEY" = #{record.withdrawMoney,jdbcType=DECIMAL},
      </if>
      <if test="record.realMoney != null">
        "REAL_MONEY" = #{record.realMoney,jdbcType=DECIMAL},
      </if>
      <if test="record.withdrawServiceCharge != null">
        "WITHDRAW_SERVICE_CHARGE" = #{record.withdrawServiceCharge,jdbcType=DECIMAL},
      </if>
      <if test="record.withdrawTime != null">
        "WITHDRAW_TIME" = #{record.withdrawTime,jdbcType=TIMESTAMP},
      </if>
      <if test="record.intoTime != null">
        "INTO_TIME" = #{record.intoTime,jdbcType=TIMESTAMP},
      </if>
      <if test="record.withdrawState != null">
        "WITHDRAW_STATE" = #{record.withdrawState,jdbcType=VARCHAR},
      </if>
      <if test="record.auditorId != null">
        "AUDITOR_ID" = #{record.auditorId,jdbcType=VARCHAR},
      </if>
      <if test="record.auditor != null">
        "AUDITOR" = #{record.auditor,jdbcType=VARCHAR},
      </if>
      <if test="record.auditTime != null">
        "AUDIT_TIME" = #{record.auditTime,jdbcType=TIMESTAMP},
      </if>
      <if test="record.remark != null">
        "REMARK" = #{record.remark,jdbcType=VARCHAR},
      </if>
    </set>
    <if test="_parameter != null">
      <include refid="Update_By_Example_Where_Clause" />
    </if>
  </update>
  <update id="updateByExample" parameterType="map">
    update USER_WITHDRAW
    set "WITHDRAW_ID" = #{record.withdrawId,jdbcType=VARCHAR},
      "WITHDRAW_ACCOUNT" = #{record.withdrawAccount,jdbcType=VARCHAR},
      "WITHDRAW_BANK" = #{record.withdrawBank,jdbcType=VARCHAR},
      "WITHDRAW_MONEY" = #{record.withdrawMoney,jdbcType=DECIMAL},
      "REAL_MONEY" = #{record.realMoney,jdbcType=DECIMAL},
      "WITHDRAW_SERVICE_CHARGE" = #{record.withdrawServiceCharge,jdbcType=DECIMAL},
      "WITHDRAW_TIME" = #{record.withdrawTime,jdbcType=TIMESTAMP},
      "INTO_TIME" = #{record.intoTime,jdbcType=TIMESTAMP},
      "WITHDRAW_STATE" = #{record.withdrawState,jdbcType=VARCHAR},
      "AUDITOR_ID" = #{record.auditorId,jdbcType=VARCHAR},
      "AUDITOR" = #{record.auditor,jdbcType=VARCHAR},
      "AUDIT_TIME" = #{record.auditTime,jdbcType=TIMESTAMP},
      "REMARK" = #{record.remark,jdbcType=VARCHAR}
    <if test="_parameter != null">
      <include refid="Update_By_Example_Where_Clause" />
    </if>
  </update>
  <update id="updateByPrimaryKeySelective" parameterType="com.share.golden.entity.UserWithdraw">
    update USER_WITHDRAW
    <set>
      <if test="withdrawAccount != null">
        "WITHDRAW_ACCOUNT" = #{withdrawAccount,jdbcType=VARCHAR},
      </if>
      <if test="withdrawBank != null">
        "WITHDRAW_BANK" = #{withdrawBank,jdbcType=VARCHAR},
      </if>
      <if test="withdrawMoney != null">
        "WITHDRAW_MONEY" = #{withdrawMoney,jdbcType=DECIMAL},
      </if>
      <if test="realMoney != null">
        "REAL_MONEY" = #{realMoney,jdbcType=DECIMAL},
      </if>
      <if test="withdrawServiceCharge != null">
        "WITHDRAW_SERVICE_CHARGE" = #{withdrawServiceCharge,jdbcType=DECIMAL},
      </if>
      <if test="withdrawTime != null">
        "WITHDRAW_TIME" = #{withdrawTime,jdbcType=TIMESTAMP},
      </if>
      <if test="intoTime != null">
        "INTO_TIME" = #{intoTime,jdbcType=TIMESTAMP},
      </if>
      <if test="withdrawState != null">
        "WITHDRAW_STATE" = #{withdrawState,jdbcType=VARCHAR},
      </if>
      <if test="auditorId != null">
        "AUDITOR_ID" = #{auditorId,jdbcType=VARCHAR},
      </if>
      <if test="auditor != null">
        "AUDITOR" = #{auditor,jdbcType=VARCHAR},
      </if>
      <if test="auditTime != null">
        "AUDIT_TIME" = #{auditTime,jdbcType=TIMESTAMP},
      </if>
      <if test="remark != null">
        "REMARK" = #{remark,jdbcType=VARCHAR},
      </if>
    </set>
    where "WITHDRAW_ID" = #{withdrawId,jdbcType=VARCHAR}
  </update>
  <update id="updateByPrimaryKey" parameterType="com.share.golden.entity.UserWithdraw">
    update USER_WITHDRAW
    set "WITHDRAW_ACCOUNT" = #{withdrawAccount,jdbcType=VARCHAR},
      "WITHDRAW_BANK" = #{withdrawBank,jdbcType=VARCHAR},
      "WITHDRAW_MONEY" = #{withdrawMoney,jdbcType=DECIMAL},
      "REAL_MONEY" = #{realMoney,jdbcType=DECIMAL},
      "WITHDRAW_SERVICE_CHARGE" = #{withdrawServiceCharge,jdbcType=DECIMAL},
      "WITHDRAW_TIME" = #{withdrawTime,jdbcType=TIMESTAMP},
      "INTO_TIME" = #{intoTime,jdbcType=TIMESTAMP},
      "WITHDRAW_STATE" = #{withdrawState,jdbcType=VARCHAR},
      "AUDITOR_ID" = #{auditorId,jdbcType=VARCHAR},
      "AUDITOR" = #{auditor,jdbcType=VARCHAR},
      "AUDIT_TIME" = #{auditTime,jdbcType=TIMESTAMP},
      "REMARK" = #{remark,jdbcType=VARCHAR}
    where "WITHDRAW_ID" = #{withdrawId,jdbcType=VARCHAR}
  </update>
</mapper>